newIDE/app/src/ObjectEditor/Editors/ShapePainterEditor.js

```diff
@@ -133,6 +136,39 @@ export default class PanelSpriteEditor extends React.Component<
             }}
           />
         </ResponsiveLineStackLayout>
+        <ResponsiveLineStackLayout alignItems="center" noMargin>
+          <Text>
+            <Trans>Anti-aliasing:</Trans>
+          </Text>
+          <SelectField
+            value={shapePainterConfiguration.getAntialiasing()}
+            onChange={(e, i, valueString: string) => {
+              shapePainterConfiguration.setAntialiasing(valueString);
+              this.forceUpdate();
+            }}
+          >
+            <SelectOption
+              key="none"
+              value="None"
+              primaryText={t`No anti-aliasing`}
+            />
+            <SelectOption
+              key="low"
+              value="Low"
+              primaryText={t`Low quality anti-aliasing`}
```

Comment:
The field name is already "anti-aliasing".
```suggestion
              primaryText={t`Low quality`}
```